packages/create-rainbowkit/templates/next-app/pages/_app.tsx

```diff
@@ -12,9 +12,11 @@ import {
   polygon,
   sepolia,
   zora,
+  Chain,
 } from 'wagmi/chains';
+import { getDefaultConfig, RainbowKitProvider } from '@rainbow-me/rainbowkit';
 
-const chains = [
+const chains: readonly [Chain, ...Chain[]] = [
```

Comment:
Is there a way to surpress these warnings without importing the type? Looks like using Wagmi directly dose:
```
export const config = createConfig({
  chains: [mainnet, sepolia],
  transports: {
    [mainnet.id]: http(),
    [sepolia.id]: http(),
  },
})
```

Reply:
yeah you can do it `as const`
```ts
const chains = [mainnet, sepolia] as const
```